tests/log-in.test.js

```diff
@@ -0,0 +1,75 @@
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+describe('Banking', () => {
+
+    it('Debería ingresar a la pagina con datos correctos', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
```

Comment:
Use **page objects** to abstract out logic regarding elements location / interaction, then make the tests interact with pages exclusively through them. That way you decouple low-level details, such as locating elements, calling the right webdrvierio method on them, etc, from actual test definition.

Ideally the flow will resamble something like this, after you create the loginPage page object:

```javascript
loginPage.open('login');
loginPage.logIn('jsmith@demo.io', 'Demo123!');
```

Basically encapsulating the complexities of loging in into methods with higher-level names

See: https://webdriver.io/docs/pageobjects/